examples/BME280_Modes/BME280_Modes.ino

```diff
@@ -17,6 +12,27 @@ Gnd (Ground)        ->  Gnd
 SDA (Serial Data)   ->  A4 on Uno/Pro-Mini, 20 on Mega2560/Due, 2 Leonardo/Pro-Micro
 SCK (Serial Clock)  ->  A5 on Uno/Pro-Mini, 21 on Mega2560/Due, 3 Leonardo/Pro-Micro
 
+
+Copyright (c) 2015-2021 Tyler Glenn, et al.
```

Comment:
I think we should keep the original year only. We can have a modified date if it is needed to keep dates for newer files or major revisions. Having 2015-2021 will always be something that needs updating.